browser/brave_news/brave_news_tab_helper.h

```diff
@@ -78,10 +91,13 @@ class BraveNewsTabHelper
   bool ShouldFindFeeds();
   void OnReceivedNewPublishers(brave_news::Publishers publishers);
   void AvailableFeedsChanged();
+  void UpdatePageFeed();
 
   raw_ptr<brave_news::BraveNewsController> controller_;
 
   std::vector<FeedDetails> rss_page_feeds_;
+  brave_news::mojom::PublisherPtr page_feed_ = nullptr;
```

Comment:
Maybe `publisher_` could be better name?

Reply:
Hmmm, I don't think publisher would make sense - a site can have multiple publishers (as it can have multiple RSS feeds). Maybe `site_feed_` would be better, or `default_feed_`?